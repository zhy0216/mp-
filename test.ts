const transpiler = new Bun.Transpiler({
	loader: "tsx",
});

const code = `
import * as whatever from "./whatever.ts"
export default function Home(props: {title: string}){
  return <node
    prop1={a.b()}
>
    <childNode prop1={a.c()} />
</node>
}`;

console.log(transpiler.transformSync(code));
console.log("==========================");
console.log(transpiler.scan(code));
console.log("==========================");
console.log(transpiler.scanImports(code));
