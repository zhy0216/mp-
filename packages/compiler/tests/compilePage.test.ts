import { test, expect } from "bun:test";


test("add", () => {
    expect(2 + 2).toEqual(4);
  });
