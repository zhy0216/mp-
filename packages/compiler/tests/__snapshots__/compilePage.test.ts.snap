// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`compile a simple page 1`] = `{}`;
